.github/scripts/check-folders.js

```diff
@@ -23,8 +21,9 @@ async function main() {
     'packagers/yarn3-workspaces-pnp/packages/sub-project-2', // We don't want to include the workspace folders in the matrix
     'platforms/aws-graviton/code', // aws-graviton doesn't have package.json at root but is included
     'platforms/m1-macstadium/code', // m1-macstadium doesn't have package.json at root but is included
+    'databases/mongodb-azure-cosmosdb', // expected to fail in failing-weekly.yaml
```

Comment:
Alternatively to listing it here, we could also add reading `failing-weekly` instead, right?

Reply:
Probably, but I suspect it's going to be the only one anyway